spyderlib/spyder.py

```diff
@@ -1196,7 +1196,7 @@ def post_visible_setup(self):
             subprocess.call(['open', app_path + MAC_APP_NAME])
 
         # Server to maintain just one Spyder instance and open files in it if
-        # the user tries to start other instances with
+        # the user tries to start other instances w

```

Comment:
Now that we're using git you can hide your error by squashing the last three commit (then force push to update the PR) if you want :)